benchexec/tablegenerator/react-table/src/tests/__snapshots__/Summary.test.js.snap

```diff
@@ -14714,7 +14714,7 @@ exports[`Render Summary for multi-table-only-columns.diff.html 1`] = `
         <div
          className="cell"
          dangerouslySetInnerHTML={"2.90"}
-         title="avg: 1.45, max: 1.49, median: 1.45, min: 1.41, stdev: 0.041"
+         title="avg: 1.45, max: 1.49, median: 1.45, min: 1.41, stdev: 0.0410"
```

Comment:
@PhilippWendler Also here: The last zero is truncated even though we have three significant digits, is this expected behaviour?

Reply:
This is probably the result of never having more than 3 decimal digits in the input in this column?

Reply:
No, it was a bug, this is now fixed on `master`.